code/modules/mob/living/carbon/human/life.dm

```diff
@@ -28,25 +28,26 @@
 	if (QDELETED(src))
 		return 0
 
-	if(.) //not dead
-		handle_active_genes()
+	if(!IsInStasis())
+		if(.) //not dead
+			handle_active_genes()
 
-	if(stat != DEAD)
-		//heart attack stuff
-		handle_heart()
+		if(stat != DEAD)
+			//heart attack stuff
+			handle_heart()
 
-	if(stat != DEAD)
-		//Stuff jammed in your limbs hurts
-		handle_embedded_objects()
+		if(stat != DEAD)
+			//Stuff jammed in your limbs hurts
+			handle_embedded_objects()
 
-	if(stat != DEAD)
-		handle_hygiene()
+		if(stat != DEAD)
+			handle_hygiene()
+
```

Comment:
am I missing something? Why is the same check here 3 times?

Reply:
Good catch, oops.

Reply:
On further reflection I believe this is in case stat changes during earlier points of Life.

Reply:
ok ill look at the individual procs to see if each check is necessary (yuck)

Reply:
still todo lmao